xyz-hub-service/src/main/java/com/here/xyz/hub/connectors/models/Connector.java

```diff
@@ -147,6 +147,11 @@ public RemoteFunctionConfig getRemoteFunction() {
    */
   public String owner;
 
+  /**
+   * An object containing the list of different params and their names that should be forwarded from Hub to the connector.
+   */
+  public ForwardParamsConfig forwardParamsConfig;
```

Comment:
That new property `forwardParamsConfig` also has to be checked by the `#equalTo()` method. Otherwise, connectors of which that property got changed, wouldn't be reloaded.

Reply:
thx, added.